recipes/cubedsphere/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cubedsphere" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cubedsphere-{{ version }}.tar.gz
+  sha256: 702884d0c57e424235f1f31634ad68a647c4410407782599aabae50994832fdc
+
+build:
+  skip: true
```

Comment:
You probably want `skip: True  # [win]`. This skips all builds. However, if this is a noarch package, we should investigate why it is not passing on Windows and either make it arch or see if we can ignore that failure.

Reply:
Hi, 

thanks for your feedback! The problem lies in one of the dependencies: https://anaconda.org/conda-forge/esmpy

I think I need to skip the windows build. 

Other than in the dependencies my package does not include any architecture specific code.

What would you suggest to change it to @ocefpaf ?

Reply:
It's always useful to check the log as the Windows build often catches other errors like missing entrypoints but in this case it can be left as noarch as Windows fails due to the `esmpy` dependency not being available. We only build noarch: python packages on Linux so it's fine if Windows fails due to missing dependencies.

Reply:
Thanks for the answer. I changed it to `skip: True  # [win]` and removed the noarch. Should I change it back?

Reply:
Yes please and add ` >=X.Y` to `python` in `host` and `run`. (2.7 or 3.6 are probably sensible values of X.Y, but check upstream if there is an explicit lower bound)

Reply:
Ok. I did this now. Ready for review @chrisburr @ocefpaf.